packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1372,8 +1372,8 @@ export const ExperimentalFontsNotEnabled = {
 export const FontFamilyNotFound = {
 	name: 'FontFamilyNotFound',
 	title: 'Font family not found',
-	message: (family: string) => `No data was found for the "${family}" family.`,
-	hint: 'This is often caused by a typo in the \`family\` property of the Font component. Check that your component is using a font family you specified in the config.',
+	message: (family: string) => `No data was found for "${family}".`,
+	hint: 'This is often caused by a typo in the \`family\` property of the Font component. Check that your component is using a \`cssVariable\` you specified in the config.',
```

Comment:
Not bad, I think. Maybe can be more direct about this being in the `<Font>` component if that’s where the error always is?

I also actually kind of like the `"${family}" family` of the old version as an extra hint at where the error is.

Maybe something like this?

```suggestion
	message: (family: string) => `No data was found for the \`"${family}"\` family passed to the \`<Font>\` component.`,
	hint: 'This is often caused by a typo. Check that your Font component is using a \`cssVariable\` specified in your config.',
```

Don’t we also need a `@description` in a JSDoc comment when using the function form for `message`?